### PR TITLE
Blog's Split App Added.

### PR DESCRIPTION
**Pull request by Tarun-Arora**
## Fixes

Fixes #[issue number] by @[issue author]

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [x] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

File: components/SiteHero.vue [hunk 1]
```diff
@@ -0,0 +1,230 @@
+<template>
+  <section :class="`hero hero-theme-${computedTheme}`">
+    <div class="banner">
+      <img
+        alt="Loading Image"
+        class="hero-bg-img"
+        :src="require(`@/assets${image}`)"
```
Comment (jyshdewangan) [inline components/SiteHero.vue:7]: Change to nuxt-img

**Comment by jyshdewangan** [comment]
1) Try dark theme for blog app
2) Make a different section for author in a particular blog.

**Review by king-11 — APPROVED** [review]

**Comment by king-11** [comment]
@all-contributors add @yash2002109 for code and bug fix

**Comment by king-11** [comment]
@all-contributors add @devansh12b2 for code and bug

**Comment by king-11** [comment]
@all-contributors add @lelouch-of-the-code  for code

**Pull request merged**